karyon-admin-web/src/main/java/netflix/adminresources/pages/PropertiesPage.java

```diff
@@ -7,7 +7,7 @@
 @AdminPage
 public class PropertiesPage extends AbstractAdminPageInfo {
 
-    public static final String PAGE_ID = "props";
+    public static final String PAGE_ID = "archprops";
```

Comment:
So for internal build- we have a separate internal version for property page (at /props). I am working on consolidating it into a single copy (its going to involve archaius ConcurrentCompositeConfiguration to be compatible with netflix configuration. Once thats done it'll be changed back to /props with single codebase running archaius based properties page